Riot/Assets/en.lproj/Vector.strings

```diff
@@ -1570,9 +1570,12 @@ Tap the + to start adding people.";
 "device_verification_self_verify_wait_new_sign_in_title" = "Verify this login";
 "device_verification_self_verify_wait_information" = "Verify this session from one of your other sessions, granting it access to encrypted messages.\n\nUse the latest %@ on your other devices:";
 "device_verification_self_verify_wait_additional_information" = "This works with %@ and other cross-signing capable Matrix clients.";
+"device_verification_self_verify_open_on_other_device_title" = "Open %@ on your other device";
+"device_verification_self_verify_open_on_other_device_information" = "You need to verify this session in order to read your secure message history.\n\nOpen Element on one of your other devices and follow the instructions.";
 "device_verification_self_verify_wait_recover_secrets_without_passphrase" = "Use Security Key";
 "device_verification_self_verify_wait_recover_secrets_with_passphrase" = "Use Security Phrase or Key";
 "device_verification_self_verify_wait_recover_secrets_additional_information" = "If you can't access an existing session";
+"device_verification_self_verify_wait_recover_secrets_additional_help" = "Can't access an existing Element session?";
```

Comment:
Adding new keys is the best way to get the right translations for these new wordings of the verification flow
but:
- I would ask you to remove the unused keys like `device_verification_self_verify_wait_recover_secrets_additional_information` (I think you can remove a key/value from all translations if the key is not use anymore)
- Do not set the application name "Element" in the string. Do what you did for `device_verification_self_verify_open_on_other_device_title`: use a parameter %@

@stefanceriu can you please confirm these 2 points?